### PR TITLE
Math Magicians - Full Website

### PR DESCRIPTION
**Pull request by mwismann**
### Math Magicians - Full Website 😎 
**The following changes were implemented:**

- Created a SPA consisting of 3 pages: ``Home``, ``Calculator``, and ``Quote`` ✅ 
- The general layout matches the wireframes ✅ 
- Added own styles to improve the look and feel ✅
- Implementation of a given ``<Route />`` for every page ✅ 
- Navigation links for all 3 routes are displayed on each page ✅ 
- Used [React Router](https://reactrouter.com/) to make the routes work ✅

### CODE REVIEW

File: src/components/Header.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
```
Comment (Lucas-Masaba) [inline src/components/Header.js]: - [x] Please do not disable eslint checks. Eslint helps improve your Javascript's quality and keeps it to standard.

File: src/index.js [hunk 2]
```diff
@@ -1,12 +1,16 @@
 import React from 'react';
 import ReactDOM from 'react-dom/client';
-import './index.css';
-import App from './App';
+import './styles/index.css';
+// eslint-disable-next-line import/no-extraneous-dependencies
```
Comment (Lucas-Masaba) [inline src/index.js]: - [x] Please do not disable eslint checks. Eslint helps improve your Javascript's quality and keeps it to standard.

File: src/App.js [hunk 3]
```diff
@@ -1,13 +1,23 @@
-import './App.css';
+// eslint-disable-next-line import/no-extraneous-dependencies
```
Comment (Lucas-Masaba) [inline src/App.js]: - [x] Please do not disable eslint checks. Eslint helps improve your Javascript's quality and keeps it to standard.

**Review by Lucas-Masaba — CHANGES_REQUESTED** [review]
# Hello @mwismann  👋 , 

Good job so far! 👍 
There are some issues that you still need to work on to go to the next project but you are almost there!

almost-there

## Highlights 🍾 
- Linter checks passing :heavy_check_mark: 
- Descriptive pull request message :scroll: 
- Your UI looks amazing :star_struck: 
- The website is working as it should :+1: 
## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions 💭 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @Lucas-Masaba in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Mathías,  

Your project is complete! 🏁 There is nothing else to say other than... it's time to merge it 🍾 Congratulations!  🎉 💯 🌟 

## Highlights 🎯 
🎲 SPA feature properly added!
🎲 Nice design!
🎲 Professional ReadMe!

Keep soaring higher! 💻  🍷  🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**